frontend/src/composables/use-playerkeys.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  onKeyStroke,
+  whenever,
+  useMagicKeys,
+  useThrottleFn
+} from '@vueuse/core';
+import playbackManager from '@/store/playbackManager';
+
+import { isTizen, isWebOS } from '@/utils/browser-detection';
+
+/**
+ * Register keyboard player control.
+ * @param osdHandler - show the player osd for a short period whenever a suitable action is called
+ */
+export function usePlayerKeys(osdHandler?: () => void): void {
+  const keys = useMagicKeys();
+
+  whenever(keys.mediapause, playbackManager.pause);
+  whenever(keys.mediaplay, playbackManager.unpause);
+  whenever(keys.mediaplaypause, playbackManager.playPause);
+  whenever(keys.mediastop, playbackManager.stop);
+  whenever(keys.mediatracknext, playbackManager.setNextTrack);
+  whenever(keys.mediatrackprevious, playbackManager.setPreviousTrack);
+  whenever(keys.audiovolumemute, playbackManager.toggleMute);
+
+  whenever(keys.space, playbackManager.playPause);
+  whenever(keys.k, playbackManager.playPause);
+  whenever(keys.m, playbackManager.toggleMute);
+
+  const callHandler = (): void => {
```

Comment:
Why this is here? This should be in the fullscreen player page, it's not something that we need to share with the rest of the players.

Reply:
I replaced it with another attempt. The blanket function is just here to prevent the `undefined` check. Would you prefer the latter?

Reply:
@endrl But why are you moving this out from the fullscreen video playback page? Unless I'm missing something, there's no reason to.

Reply:
Can you please clarify about what  code lines we are talking? The current behaviour is that `video/index || audio/index || AudioControls` is mounted (but never at the same time). They all get the same key bindings and the video player has a callback fn to show the osd with a timeout when you press forward/backward.